api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -296,4 +298,25 @@
 	public static final String REVERSE_INCLUDE_SEARCH_HANDLER = "_revinclude.search.handler";
 	
 	public static final String CONDITION_OBSERVATION_CONCEPT_UUID = "1284AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+	
+	public static final String FHIR_MEDIA = "media";
+	
+	public static final String MEDIA_STATUS = "media.status";
+	
+	public static final String MEDIA_TYPE = "media.type";
+	
+	public static final String MEDIA_SUBJECT = "media.subject";
+	
+	public static final String MEDIA_ENCOUNTER_REFERENCE = "media.encounter.reference";
+	
+	public static final String MEDIA_CREATED_ON = "media.created.on";
+	
+	public static final String MEDIA_CONTENT_TYPE = "media.content.type";
+	
+	public static final String CONTENT_DATA = "content.data";
+	
+	public static final String CONTENT_TITLE = "content.title";
+	
+	public static final String CONTENT_DATE_OF_CREATION = "content.creation";
+	
```

Comment:
We probably don't need all these constants, since we're generally not parsing the JSON ourselves. We usually just need the resource name as a constant, e.g., `Media`